lib/settings/settings.dart

```diff
@@ -177,6 +177,8 @@ class Settings extends ChangeNotifier with SettingsSharedPref {
 }
 
 class NoteFileNameFormat extends GjSetting {
+  final bool usesTitle;
```

Comment:
introducing this here to prevent e.g. this bug https://github.com/GitJournal/GitJournal/issues/654 from happening again.